.github/workflows/pylint.yml

```diff
@@ -0,0 +1,42 @@
+name: Pylint
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
```

Comment:
Should we install the dependencies mentioned in the requirement.txt?

Reply:
It's not necessary in order to lint. We just need the linter and the source files.